packages/System-Changes.package/.filetree

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Everything below that point has nothing to do with our project and is only displayed here because we had to add System-Changes to our tracked packages because of our extension for the ChangeRecord class.